### PR TITLE
LVIIICNET-150/LVIIICNET-112 News frontend features

### PR DESCRIPTION
**Pull request by Rudikk**

### CODE REVIEW

File: TheraLang.Web/ClientApp/src/njson-typings.d.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+declare module "*.json" {
+    const value: any;
+    export default value;
+  }
```
Comment (mod1fy) [inline TheraLang.Web/ClientApp/src/njson-typings.d.ts]: Do we need this?

File: TheraLang.Web/ClientApp/src/assets/mock-responses/test-preview-news.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        },
+        {
+            "id": "3",
+            "title" : "Test 3",
+            "text" : "Text 3",
+            "mainImageUrl" : null,
+            "authorName" : "Bodya",
+            "createdDateUtc" : "2019-05-08"
+        }
+    ]
```
Comment (mod1fy) [inline TheraLang.Web/ClientApp/src/assets/mock-responses/test-preview-news.json]: Maybe remove mock

File: TheraLang.Web/ClientApp/src/app/modules/main/pages/news/news-page.component.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<button mat-raised-button class="create-button" color = "primary" [routerLink]="['/news/create']" *ngIf="isAuthenticated()">
```
Comment (mod1fy) [inline TheraLang.Web/ClientApp/src/app/modules/main/pages/news/news-page.component.html]: You can use userService.isAuthinticated()

File: TheraLang.Web/ClientApp/src/njson-typings.d.ts (hunk 1 above)
Comment (Rudikk) [reply TheraLang.Web/ClientApp/src/njson-typings.d.ts]: No, it's just a trash, I created this when was testing news output

Comment (mod1fy) [reply TheraLang.Web/ClientApp/src/njson-typings.d.ts]: Remove?

Comment (Rudikk) [reply TheraLang.Web/ClientApp/src/njson-typings.d.ts]: Yes, I removed, and also did some fixes you described below

File: TheraLang.Web/ClientApp/src/app/modules/main/toolbar/toolbar.component.html [hunk 4]
```diff
@@ -5,8 +5,8 @@
         <img src="assets/img/uttmm.png" alt="Home" class="img-wrapper">
       </a>
 
-      <app-cms-pages-toolbar-item></app-cms-pages-toolbar-item>
-
+      <app-cms-pages-toolbar-item *ngIf="isAuthenticated()"></app-cms-pages-toolbar-item>
```
Comment (mod1fy) [inline TheraLang.Web/ClientApp/src/app/modules/main/toolbar/toolbar.component.html]: remove isAuthenticated

**Pull request merged**